src/Orc.Controls/Controls/StepBar/Models/Interfaces/IStepBarItem.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Orc.Controls
+{
+    public interface IStepBarItem
+    {
+        string Title { get; set; }
```

Comment:
The interface does not need the setters (it will be model dependant) so the setters can be removed.